### DISCUSSION TITLE
Themes not loading

### DISCUSSION THREAD

**Discussion by nlordell** [community/community#3618] [Codespaces] [answered]
When opening up a newly created Codespace, I noticed that the the icons in the file browser were missing, as well as syntax highlighting.

Upon devtools inspection, I noticed that various requests to `https://*.github.dev/assets/*/*/vscode-remote-resource?path=*` requests were failing with 404 errors, among them the SETI icon theme and VSCode light theme (which explains the issues ☝️).

**edit**: Some further testing indicates this is an issue with Codespaces under Firefox. There are no issues with Chromium or Safari.

**Comment by 2percentsilk** [comment, marked as answer]
@nlordell

Thanks for letting us know and we're sorry you ran into those issues. Firefox isn't currently a fully supported browser for Codespaces at this time - Chromium is our recommended browser for best experience. I'll see what we can do about making that clearer in our documentation/in-product!
